controllers/tab/workflow/WorkflowTabHandler.inc.php

```diff
@@ -44,7 +139,27 @@ function fetchTab($args, $request) {
 					__('editor.article.schedulePublication')
 				);
 				$templateMgr->assign('schedulePublicationLinkAction', $schedulePublicationLinkAction);
+
+				$schedulePublicationLinkAction = $this->displaySchedulePublicationLinkAction($request);
+				$templateMgr->assign('schedulePublicationLinkAction', $schedulePublicationLinkAction);
+
+				$xmlFileCreateLinkAction = $this->xmlFileCreateLinkAction($request);
+				$templateMgr->assign('xmlFileCreateLinkAction', $xmlFileCreateLinkAction);
+
+				$xmlFileUploadLinkAction = $this->xmlFileUploadLinkAction($request);
+				$templateMgr->assign('xmlFileUploadLinkAction', $xmlFileUploadLinkAction);
+
+
+				// Get if jatsTemplatePlugin is enabled
+
+				$pluginSettingsDao = DAORegistry::getDAO('PluginSettingsDAO');
+				$context = $request->getContext();
+				if ($pluginSettingsDao->settingExists($context->getId(), 'jatstemplateplugin', 'enabled')) {
```

Comment:
It would be better not to code anything specific to JATSTemplate into the core codebase; it breaks plugin encapsulation.